services/orchest-webserver/client/src/pipeline-view/hooks/useAutoStartSession.tsx

```diff
@@ -48,27 +48,27 @@ export const useAutoStartSession = ({ isReadOnly = false }) => {
         pipelineUuid,
         BUILD_IMAGE_SOLUTION_VIEW.PIPELINE
       );
-      if (
+
+      const isBuildingJupiterEnvironment =
```

Comment:
typo

```suggestion
      const isBuildingJupyterEnvironment =
```

(would also require a change of the variable below)

Reply:
Fixed. :+1: